script/incremental_build.sh

```diff
@@ -55,8 +55,5 @@ else
   else
     echo running "${ACTIONS[@]}"
     (cd "$REPO_DIR" && plugin_tools "${ACTIONS[@]}" --plugins="$CHANGED_PACKAGES" --exclude="$ALL_EXCLUDED" ${PLUGIN_SHARDING[@]})
-    echo "Running version check for changed packages"
```

Comment:
@blasten Did you remember why this check was removed?

Reply:
It looks like it was a mistake when I merged the nnbd branch

Reply:
KK, so this PR should re-enable this check.